doc/changes/changes_1.7.0.md

```diff
@@ -1,32 +1,58 @@
-# Exasol Kafka Connector Extension 1.7.0, released 2023-??-??
+# Exasol Kafka Connector Extension 1.7.0, released 2023-10-17
 
-Code name:
+Code name: Extension manager support
 
 ## Summary
 
+Adds extension manager support.
+
 ## Features
 
-* ISSUE_NUMBER: description
+* 72: Adds extension manager support.
+
+* Excluded the following CVEs in test dependencies:
+    * io.netty:netty-handler:jar:4.1.95.Final:test
+        * CVE-2023-4586 
+    * fr.turri:aXMLRPC:jar:1.13.0:test
+        * CVE-2020-36641
+    * org.eclipse.jetty:jetty-http:jar:9.4.51.v20230217:test
+        * CVE-2023-40167
+    * org.eclipse.jetty:jetty-servlets:jar:9.4.51.v20230217:test
+        * CVE-2023-36479
+    * org.xerial.snappy:snappy-java:jar:1.1.10.1:runtime; vulnerability under org.testcontainers:kafka:jar:1.19.0:test
+        * CVE-2023-43642
+    * org.eclipse.jetty.http2:http2-common:jar:9.4.51.v20230217:test
+        * CVE-2023-44487
+    * org.apache.avro:avro:jar:1.9.2:test 
+        * CVE-2023-39410
+    * org.eclipse.jetty.http2:http2-hpack:jar:9.4.51.v20230217:test
+        * CVE-2023-36478
+    * org.apache.zookeeper:zookeeper:jar:3.6.3:test
+        * CVE-2023-44981
+    * org.json:json:jar:20230227:test
+        * CVE-2023-5072
```

Comment:
Are there versions available that fix these vulnerabilities?

Reply:
I've updated everything I could in the project itself. I'll have a closer look at our own depedencies down the line, see what we can update.